File: EducationApp.BusinessLogicLayer/Models/PrintingEditions/PrintingEditionModel.cs
﻿using EducationApp.BusinessLogicLayer.Models.Base;
using System.Collections.Generic;

namespace EducationApp.BusinessLogicLayer.Models.PrintingEditions
{
    public class PrintingEditionModel : BaseModel
    {
        public ICollection<PrintingEditionModelItem> Items { get; set; } = new List<PrintingEditionModelItem>();
        public int Count { get; set; } 
    }
}

File: ClientApp/src/app/shared/models/printing-editions/PrintingEditionModel.ts
import { PrintingEditionModelItem } from 'src/app/shared/models/printing-editions/PrintingEditionModelItem';
import { BaseModel } from 'src/app/shared/models/Base/BaseModel';

export class PrintingEditionModel extends BaseModel {

    count: number;
    items: Array<PrintingEditionModelItem>;

}